src/test/test_verify_samples.cpp

```
@@ -0,0 +1,314 @@
+    bool matched = false;
+    for (auto& raw_prog : raw_progs) {
+        if (entry.count != 1 && raw_prog.function_name != entry.function) {
+            continue;
+        }
+        matched = true;
+        INFO("function_name=" << raw_prog.function_name);
+        if (entry.xfail) {
+            INFO("issue_kind=" << verify_test::to_string(entry.kind));
+            INFO("reason=" << entry.reason);
+        }
+
```

Cppcheck (2.21.0)

[performance] Function parameter 'premise_set' should be passed by const reference.

(passedByValue)